frontend/src/components/request/index.ts

```diff
@@ -0,0 +1,4 @@
+import RequestErrorModal from './RequestErrorModal';
+import RequestErrorSnackbar from './RequestErrorSnackbar';
+
+export { RequestErrorModal, RequestErrorSnackbar };
```

Comment:
do you think it would be a good idea to export snackBarAlerts here as well?

Reply:
I think we don't need to. My reasoning as to why I made a separate modal and snackbar for components/request is that since request components are only meant to be used for API calls, so I didn't put RequestErrorSnackbar in the general snackbar folder and same for RequestErrorModal as well. I just leave them here bc they're really specific to request and is not supposed to be used outside of rendering modal/snackbar based on whether an error occurred for a request or not. Since snackbarAlerts can be used for a lot of purposes, it is better imo to leave it where it is.

Reply:
Sounds good